packages/expo-cli/src/commands/utils/validateDependenciesVersions.ts

```diff
@@ -8,12 +8,14 @@ import { Versions } from 'xdl';
 
 import CommandError from '../../CommandError';
 import Log from '../../log';
+import { actionAsync } from '../installAsync';
```

Comment:
Generally you shouldn't use the `actionAsync` function directly since it's meant to be used from commander, using the internal functions would potentially be better for this use case.

Reply:
I've looked at that `actionAsync` method, and it happens to do exactly what it needs to do here, as we're doing the _exact_ equivalent of calling `expo install package1 package2 package3` here. `actionAsync` directly contains the entire implementation flow for doing that, using the internal functions directly would mean duplicating/copying that entire logic, or extracting it into another exported function that's not `actionAsync`, but `doTheActualWorkAsync`, which is more of a semantic workaround than much else.